docs/markdown/Release-notes-for-0.42.0.md

```diff
@@ -11,3 +11,8 @@ short-description: Release notes for 0.42 (preliminary)
 
 Creating distribution tarballs can now be made out of projects based on
 Mercurial. As before, this remains possible only with the Ninja backend.
+
+## Keyword argument verification
+
+Meson will now check the keyword arguments used when calling any function
+and error out if any of keyword arguments is not known.
```

Comment:
*any of the keyword arguments are not known.

Also, we should not error out right now, we should only warn with a message that this will become a hard error later. No need to break people's already-working build files for no reason.